src/app/content/hooks/locationChange.spec.ts

```diff
@@ -203,11 +203,13 @@ describe('locationChange', () => {
       publish_date: '2012-06-21',
     };
 
+    mockConfig.BOOKS[mockOtherBook.id] = {defaultVersion: mockOtherBook.version};
+    jest.doMock('../../../config', () => mockConfig);
+
```

Comment:
I think you could just `mockImplementation` on `getBookVersionFromUUID`

Reply:
Is it really needed? It seems easier to mock the config there.

Reply:
i agree that it would be more straight forward to use a spy on `getBookVersionFromUUID`, but its fine like this